src/test/java/uk/gov/companieshouse/uri/web/transformer/CompanyDetailsTransformerImplTest.java

```diff
@@ -193,6 +198,39 @@ void profileApiToDetailsMissingChargesLink() {
         assertFalse(companyDetails.hasCharges());
     }
     
+    @Test
+    void profileApiToDetailsOverseaCompanyWithOriginatingRegistryPresent() {
+        CompanyProfileApi companyProfileApi = populatedCompanyProfileApi();
+        companyProfileApi.setForeignCompanyDetails(foreignCompanyDetails(true, true));
```

Comment:
I am wondering if the use of the method foreignCompanyDetails() in this test class might be clearer if there was something in the name to show it is a factory method? Maybe something like getForeignCompanyDetails() or makeForeignCompanyDetails()

Reply:
I've changed the name to createForeignCompanyDetailsApi() - that does make it clearer.

Reply:
Yes thanks